defaults/main.yml

```diff
@@ -20,8 +20,8 @@ k8s_echotest_letsencrypt_issuer: 'letsencrypt-staging'
 
 k8s_papertrail_logspout_enabled: "{{ k8s_papertrail_logspout_destination | length > 0 }}"
 k8s_papertrail_logspout_destination: ''  # format: syslog+tls://logsN.papertrailapp.com:XXXXX
-k8s_papertrail_logspout_syslog_hostname: '{% raw %}{{ index .Container.Config.Labels "io.kubernetes.container.name" }}{% endraw %}'
-k8s_papertrail_logspout_syslog_tag: '{% raw %}{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}[{{ index .Container.Config.Labels "io.kubernetes.pod.name" }}]{% endraw %}'
+k8s_papertrail_logspout_syslog_hostname: '{% raw %}{{ index .Container.Config.Labels "io.kubernetes.pod.namespace" }}{% endraw %}'
```

Comment:
Cool!

I like how this organizes the systems on Papertrail for the app namespaces, but I think we run into collisions for the other pods, like ``cert-manager``, that live in common namespaces. I think ``default`` is currently for ``pressweb`` now:

https://my.papertrailapp.com/systems/default/events

This search shows the comparison between saguaro and pressweb:

https://my.papertrailapp.com/events?q=program%3Anewrelic-infra%20default